README.md

```diff
@@ -24,22 +24,36 @@ const Example = () => (
       substitleClass=""
       position="top"
     />
-    <PlotLineSeries
-      data={{ x: [0, 1, 2, 3, 4, 5], y: [0, 1, 2, 3, 3, 3] }}
```

Comment:
we would like to keep this way of defining the data. I'm sure it's possible to work with this in d3

Reply:
It's possible, but is not meant to be used like that, so will be more expensive to internally change the format than use https://github.com/mljs/xy-convert in the parent